### PR TITLE
docs: Change important alert of ngFor

### PR DESCRIPTION
**Pull request by bampakoa**
Update the important alert of `ngFor` in the **Displaying Data** guide so that it has a unique format with that of `ngIf`.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/displaying-data.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ It marks that `<li>` element (and its children) as the "repeater template":
 <div class="alert is-important">
 
 Don't forget the leading asterisk (\*) in `*ngFor`. It is an essential part of the syntax.
-For more information, see the [Template Syntax](guide/template-syntax#ngFor) page.
+Read more about `ngFor` and `*` in the [ngFor section](guide/template-syntax#ngfor) of the [Template Syntax](guide/template-syntax) page.
```
Comment (gkalpak) [inline aio/content/guide/displaying-data.md]: Good idea :+1: However, in contrast to the `ngIf` section, the `ngFor` section does not contain info about the `*`.
Maybe we could add an alert similar to [this one][1]?

[1]: https://github.com/angular/angular/blame/528e25a81a90c828365a2df08328f55be01e421b/aio/content/guide/template-syntax.md#L1528-L1534

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx, @bampakoa!

File: aio/content/guide/displaying-data.md (hunk 1 above)
Comment (bampakoa) [reply aio/content/guide/displaying-data.md]: @gkalpak you are right 👌 You mean to add the alert in the `ngFor` section of the template syntax, yes?

Comment (gkalpak) [reply aio/content/guide/displaying-data.md]: Yes :+1:

**Review by gkalpak — APPROVED** [review]
Thx, @bampakoa - LGTM 👌

**Comment by gkalpak** [comment]
**merge-assistance:** Global approval for this docs-only change.

**Pull request closed without merging**